runner/run.go

```diff
@@ -257,7 +258,7 @@ func markAndFlush(runContext *TestRunContext, dest *ftwhttp.Destination, stageID
 	return nil, fmt.Errorf("can't find log marker. Am I reading the correct log? Log file: %s", runContext.Config.LogFile)
 }
 
-func needToSkipTest(include *regexp.Regexp, exclude *regexp.Regexp, testCase *schema.Test) bool {
+func needToSkipTest(include *regexp.Regexp, exclude *regexp.Regexp, tag *regexp.Regexp, testCase *schema.Test) bool {
```

Comment:
Change the signature of the method so that it simply accepts the run context as parameter.